ports/servoshell/minibrowser.rs

```diff
@@ -217,6 +220,23 @@ impl Minibrowser {
                 return;
             };
             let mut embedder_events = vec![];
+
+            if self.hovered_link.is_some() {
+                TopBottomPanel::bottom("innner_hover_links")
+                    .max_height(0.0)
+                    .show(ctx, |ui| {
```

Comment:
this can be done without a TopBottomPanel by replacing `ui.cursor().max.y` with `ctx.available_rect().max.y`